pkg/scheme/qserv/statefulset.go

```diff
@@ -298,7 +298,7 @@ func GenerateXrootdStatefulSet(cr *qservv1alpha1.Qserv, labels map[string]string
 
 	labels = util.MergeLabels(labels, util.GetLabels(constants.XrootdRedirectorName, cr.Name))
 
-	var replicas int32 = 2
+	var replicas int32 = cr.Spec.Xrootd.Replicas
```

Comment:
I guess this is not case sensitive? It still might be better to make it match 🤷‍♂